cloud-services/mbl-cloud-client/scripts/arm_update_activate.sh

```diff
@@ -195,18 +247,38 @@ ewuc_component_filename="$2"
         exit 60
     fi
 
-    # Linux always considers the sector size to be 512 bytes, no matter what
-    # the device's actual block size is. We just let dd use its default block size and
-    # ensure the seek is always a byte count.
-    # See: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/linux/types.h?id=v4.4-rc6#n121
-    if ! dd if="$UPDATE_PAYLOAD_DIR/$ewuc_component_filename" of="$ewuc_disk_name" oflag=seek_bytes conv=fsync seek="$ewuc_offset_bytes"; then
-        printf "Writing %s to disk failed.\n" "$ewuc_component_filename"
-        exit 61
+    # Copy files to the boot partition, and the bootloader FS partition if it exists
+    if [ ! -z "$ewuc_boot_part_mnt_point"  ]; then
```

Comment:
That's because currently on rpi3 we duplicate the contents of the blfs partition in the boot1 partition. The FIT image really lives in the boot partition but we haven't yet split up the contents of `IMAGE_BOOT_FILES` to put different files on blfs to boot1 (or configured U-Boot to look for the FIT image on the boot1 partition rather than the blfs partition).

Reply:
Although this deserves a comment. I'll add one.

Reply:
https://github.com/ARMmbed/mbl-core/pull/208/commits/bdc918766a43ac135e5445e963dea76684b8e2eb